mono/metadata/mono-debug.h

```diff
@@ -10,6 +10,7 @@
 #include <mono/utils/mono-publib.h>
 #include <mono/metadata/image.h>
 #include <mono/metadata/appdomain.h>
+#include <mono/metadata/loader-internals.h>
```

Comment:
mono-debug.h is a public header so we can't include internals here (see build failure in Mono iOS SDK job)